### ISSUE TITLE
Incorporating youtube search

### ISSUE THREAD

**Issue by mraza007** [mraza007/videodownloader#13]
instead of copying and pasting URL. We should create a search field that will allow us to search videos right from the gui or cli

**Comment by jslay88** [comment]
I'd say it should probably be pretty low priority, as YouTube already has a great search (don't want to reinvent the wheel), and most people will want to initially watch what they are about to download, before downloading it.

If anything, a search feature should only be included if we can effectively also allow people to watch the video within the application as well. Searching via cli I think wouldn't be worth our efforts, for the previously mentioned reasons.

**Comment by mraza007** [comment]
We can assign this as low priority. I was thinking about implementing it in gui so people can search and people can play right in the gui

**Comment by jslay88** [comment]
yeah, was looking into pyqt instead of tkinter for a more advanced interface, as there are serious limitations behind tkinter.

**Comment by mraza007** [comment]
I see Kivy is pretty cool framework 
yup and you are right tkinter won't be able to fit all feature we want to add btw the thumbnail was pretty cool
